main/src/ca/uwaterloo/flix/language/phase/unification/BddFormula.scala

```diff
@@ -15,10 +15,105 @@
  */
 package ca.uwaterloo.flix.language.phase.unification
 
+import ca.uwaterloo.flix.language.ast.{SourceLocation, Symbol, Type}
+import ca.uwaterloo.flix.util.InternalCompilerException
+import ca.uwaterloo.flix.util.collection.Bimap
+
+import scala.collection.immutable.SortedSet
+import org.sosy_lab.pjbdd.api.{Builders, Creator, DD}
+
 object BddFormula {
 
-  class Bdd
+  //Thread-safe factory for creating BDDs
```

Comment:
Comments associated with entities (e.g. vals) are:

```
/**
 * A thread-safe factory for creating BDDs (using PJBDD).
 */
```

 A single comment to document some code is just `//` and if its an important step we sometimes write:

```
//
// Here something new happens!
//
```